connectivity/check/deployment.go

```diff
@@ -413,6 +572,31 @@ func (ct *ConnectivityTest) validateDeployment(ctx context.Context) error {
 		}
 	}
 
+	perfPods, err := ct.client.ListPods(ctx, ct.params.TestNamespace, metav1.ListOptions{LabelSelector: "kind=" + kindPerfName})
+	if err != nil {
+		return fmt.Errorf("unable to list perf pods: %s", err)
```

Comment:
A few more instances of `%s` that need to be `%w`

Reply:
Found a couple other places in the code... amending those changes... Thanks again @christarazi